.github/renovate.json

```diff
@@ -12,5 +12,10 @@
       "matchPackageNames": ["com.google.protobuf"],
       "allowedVersions": "<=0.8.19"
     }
-  ]
+  ],
+  "postUpgradeTasks": {
```

Comment:
It seems like this will work only on self-hosted instances:

> Post-upgrade tasks can only be used on self-hosted Renovate instances.
> https://docs.renovatebot.com/configuration-options/#postupgradetasks

Reply:
Bummer :cry:

Reply:
Rebased, this diff is obsolete.